app/src/main/res/values/strings.xml

```diff
@@ -10,6 +10,12 @@
     <string name="title_offboard_router_java">Offboard Router Java</string>
     <string name="description_offboard_router_java">Shows routes using offboard routing api in Java code.</string>
 
+    <string name="title_onboard_router_kotlin" translatable="false">Onboard Router Kotlin</string>
```

Comment:
```suggestion
    <string name="title_onboard_router_kotlin" translatable="false">Onboard Router %s</string>
```

and String.format for `java` and `kotlin`?
the same with lines below

Reply:
discussed